sql/core/src/test/scala/org/apache/spark/sql/execution/joins/BroadcastJoinSuite.scala

```diff
@@ -48,7 +48,7 @@ class BroadcastJoinSuite extends QueryTest with SQLTestUtils {
       .setMaster("local-cluster[2,1,1024]")
       .setAppName("testing")
     val sc = new SparkContext(conf)
-    spark = SparkSession.builder.getOrCreate()
```

Comment:
Oh, I checked my PR again and found that I already fix this in my PR.
Yes, right. You had better revert this line to avoid unnecessary conflicts.